src/servers_non_y2/ag_etc_sudoers

```diff
@@ -3,6 +3,11 @@
 # Author: Bubli <kmachalkova@suse.cz>
 #
 # An agent for parsing /etc/sudoers file
+#
+# TODO: stop ignoring the configuration from directives like "#includedir /etc/sudoers.d" which are
+# being processed as a comment since the agent does not understand them for now.
+# FIXME: "$previous_content", formerly "$comment", should not be "associated" to a rule or alias
```

Comment:
actually this association is often fine as it is related to that key ( at least part of it ), but issue is that directives, that is not plain comments that can be easily moved or deleted.

Reply:
> actually this association is often fine

Right, but the problem is the _often_ part :disappointed: 

> but issue is that directives, that is not plain comments that can be easily moved or deleted.

I'm going to reword the _FIXME_ note a bit after merge the PR